pkg/purl/purl.go

```diff
@@ -290,6 +290,8 @@ func purlType(t string) string {
 		return packageurl.TypeMaven
 	case string(analyzer.TypeBundler), string(analyzer.TypeGemSpec):
 		return packageurl.TypeGem
+	case string(analyzer.TypeNuget), string(ftypes.DotNetCore):
```

Comment:
It should be `TypeDotNetDeps`, no?

```suggestion
	case string(analyzer.TypeNuget), string(analyzer. TypeDotNetDeps):
```

Reply:
Also, I think we need to add `TypeNuget` here.
https://github.com/aquasecurity/trivy/blob/4c32b47548441dd31beba8a3e91871fd94a0ec09/pkg/purl/purl.go#L80-L98

Reply:
Done